index.html

```diff
@@ -0,0 +1,193 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <header class="logo">
+        <a href="#"><h3>KGW</h3></a>
+        <nav>
+            <img src="image/Union.png" alt="menu" class="menu">
+            <ul>
+                <li> <a href="#">Portifolio</a> </li>
+                <li> <a href="#">About</a> </li>
+                <li> <a href="#">contact</a> </li>
+                <li> <a href="#"><img src="images/Union.png" alt="bars"></a> </li>
+            </ul>            
+        </nav>
+    </header>
+<section class="content">
+        <h1>Kidde George</h1>
+        <h2>Software Developer</h2>
+        <p>I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,  don’t hestiate to contact me.</p>
+</section>
+<div class="img">
+    <img src="images/Vector4.png" alt="line">
```

Comment:
- [ ] Kindly give this image a width: 100% and remove the padding as her width is too long and it causes a horizontal scroll.